timestamp-impl/src/main/java/com/palantir/timestamp/PersistentUpperLimit.java

```diff
@@ -32,7 +32,9 @@
     private volatile long lastIncreasedTime;
 
     public PersistentUpperLimit(TimestampBoundStore tbs, Clock clock, TimestampAllocationFailures allocationFailures) {
-        DebugLogger.logger.info("Creating PersistentUpperLimit object on thread {}. This should only happen once.",
+        DebugLogger.logger.info("Creating PersistentUpperLimit object on thread {}. This should only happen once."
```

Comment:
nit: I think the `This should only happen once.` here should be removed (probably hanging from copypaste)